src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -89,6 +88,18 @@ class PgAutoscaler(MgrModule):
             "desc": "report on pool pg_num sizing recommendation and intent",
             "perm": "r"
         },
+        {
+            "cmd": "osd pool set autoscale-profile scale-up",
+            "desc": "set the autoscaler behavior to start out with minimum pgs\
```

Comment:
for multiline strings python is a bit tricky - written this way, the indentation gets included in the string as whitespace.

To avoid this you can use a new string on each line, like this:

```"desc": "foo"\
                "bar",
```

or add parens around them to make it even clearer that it's a single string spread across lines without the backslashes:

```"desc": ("foo"
                "bar"),
```

this applies to these commands and the autoscale_profile description below

Reply:
thank you for the tip